### PR TITLE
PR: Feature/play screen design

### PR DESCRIPTION
**Pull request by roboshoes**
This Pull Requests adds most parts of the new design for the play screen. I sent a PR dispite it not being complete because it adds a few globally used mixins and features that I want for others to use as a base.

What is missing: The speedometer and the Highscore button. 
But in a matter of rapid development I want to push the first result into the development branche.

Please review.

### CODE REVIEW

**Comment by tombu** [comment]
In general: Nice work!

Stuff that I don´t like that much:
- the background pattern feels cold - the old one had a much warmer green
- the 'other track' button doesn´t work for me
- the current font does look aweful when displaying the race times in the highscores
- the clipboard (and the track a the bottom-left on the HomeScreen) is bigger than the screen :(
- maybe we should rework the clock - the digits look nice but the box does no longer match our new style
- the top bar looks like it is below the gras (maybe this is an optical illusion caused by the white line) - Can we please make that edge smoother?

File: app/assets/javascripts/slotcars/play/views/result_view.js.coffee [hunk 1]
```diff
@@ -11,6 +11,9 @@ Play.ResultView = Ember.View.extend
     @onLapTimesChange()
     @onRaceTimeChange()
 
+  onRestartClick: ->
+    @gameController.restartGame()
```
Comment (tombu) [inline app/assets/javascripts/slotcars/play/views/result_view.js.coffee]: This does not work on my machine ... and please make it a one-liner :wink:

**Comment by DominikGuzei** [comment]
@tombu nice fisting indeed! :wink:

**Comment by tombu** [comment]
If I don´t say it now I´ll have to change it my self afterwards ... :wink:

**Comment by tombu** [comment]
You satisfied me :wink: thanks! :+1: for merge

**Comment by DominikGuzei** [comment]
One minor graphical bug I saw: The first red light changes the graphics (edge gets lighter) as soon as the third red light is on! but its just a minor thing that can also be fixed after the presentation ;-) :+1: for merge!

**Pull request merged**